### PR TITLE
doc: cookbook: add article on how to add a frontend library

### PR DESCRIPTION
**Pull request by fscherf**
Signed-off-by: Florian Scherf <f.scherf@pengutronix.de>

### CODE REVIEW

**Comment by fscherf** [comment]
@laundmo,  @steezeburger: Any thoughts?

File: doc/content/cookbook/integrating-a-frontend-library.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+                lona_context.setup();
+            </script>
+        </body>
+    </html>
+
+.. code-block:: python
+
+    from lona import LonaView
+    from lona.html import HTML, H1, Div
```
Comment (korantu) [inline doc/content/cookbook/integrating-a-frontend-library.rst:112]: Are `H1` and `Div` needed here?

**Review by korantu — COMMENTED** [review]
Looks good to me

File: doc/content/cookbook/integrating-a-frontend-library.rst (hunk 1 above)
Comment (fscherf) [reply doc/content/cookbook/integrating-a-frontend-library.rst]: Nope :) Thanks for noticing :)

**Review by korantu — APPROVED** [review]

**Review by korantu — APPROVED** [review]

**Comment by fscherf** [comment]
I am sorry that it took me so long. @maratori added new API to add static files, which will be in the next release. Before the release i will finish this article and release everything together.

**Comment by fscherf** [comment]
@korantu Thanks for reviewing, and for your patience :)

**Pull request merged**